build.cmd

```diff
@@ -403,6 +403,7 @@ set "__BuildLog=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.l
 set "__BuildWrn=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.wrn"
 set "__BuildErr=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.err"
 set __msbuildLogArgs=^
+/p:UseOpenSourceSign=false ^
```

Comment:
@joperezr can you work with @chcosta to update the VSO build definitions to set this property.